File: Praktikum C/PRAK205-2210817210033-HariOctavianDelrossi.c
#include <stdio.h>
#include <math.h>
int main(){
    int A,B,C,K,L;

    printf("Masukkan nilai A: ");scanf("%d",&A);
    printf("Masukkan nilai B: ");scanf("%d",&B);
    printf("\n");

    C=sqrt((B*B)-(A*A));
    printf("Alas= %d cm\n",C);

    printf("Tinggi= %d cm\n",A);

    K=A+B+C;
    printf("Keliling= %d cm\n",K);

    L=0.5*C*A;
    printf("Luas= %d cm^2",L);
    return 0;
}

File: Praktikum Python/PRAK205-2210817210033-HariOctavianDelrossi.py
import math

A=int(input('Masukkan nilai A: '))
B=int(input('Masukkan nilai B: '))

##Mencari Alas
C=math.sqrt((B*B)-(A*A))
print('\nAlas= ',end='')
print('%d' %C, 'cm')

##Mencari Tinggi
print('Tinggi=',A,'cm')

##Mencari Keliling
K=A+B+C
print('Keliling= ',end='')
print('%d' %K, 'cm')

##Mencari Luas
L=0.5*C*A
print('Luas= ',end='')
print('%d' %L, 'cm^2')